python/paddle/incubate/__init__.py

```diff
@@ -16,3 +16,51 @@
 __all__ += ["reader"]
 
 from ..fluid.contrib import reader
+from ..fluid import framework
+from . import complex
+
+
+def monkey_patch_math_complex():
```

Comment:
recommended to create a new *.py in the incubate/complex directory to write this code, it is not recommended to put complex logic in __init__.py

Reply:
only call `monkey_patch_math_complex()` in __init__.py  is better, add call it in `incubate/complex/__init__.py` is better?

Reply:
create a new py file,  mv monkey_patch_math_complex() into incubate/complex/__init__.py

Reply:
Done